argocli/rockcraft.yaml

```
@@ -0,0 +1,90 @@
+name: argo-cli
+summary: "Argo Server rock"
+description: "Container-Native Workflow Engine for Kubernetes"
+version: "v3.3.8_1"
```

Need to bring version up to spec: <upstream_version>_<base_image_version>_<rock_version>
```suggestion
version: "v3.3.8_22.04_1"
```

---

LGTM otherwise.